db/v-1/insert-test-data/project-insert.xml

```diff
@@ -9,13 +9,13 @@
         <insert dbms="mysql" tableName="project">
             <column name="id" value="1" />
             <column name="name" value="huaweiName" />
-            <column name="url" value="huawei.com" />
+            <column name="url" value="https://huawei.com" />
             <column name="description" value="test description" />
         </insert>
         <insert dbms="mysql" tableName="project">
             <column name="id" value="2" />
-            <column name="name" value="huaweiName 2" />
```

Comment:
What's wrong with whitespaces?

Reply:
We want to allow only alphanumeric symbols and hyphens, so it is better to have test data valid